components/raftstore/src/store/worker/pd.rs

```diff
@@ -775,7 +776,7 @@ where
     pub fn maybe_send_read_stats(&self, read_stats: ReadStats) {
         if let Some(sender) = &self.read_stats_sender {
             if sender.send(read_stats).is_err() {
-                warn!("send read_stats failed, are we shutting down?")
+                debug!("send read_stats failed, are we shutting down?")
```

Comment:
better not change?

Reply:
The channel is changed to bounded `sync_channel`, it may overwhelm log files when there are lots of concurrent coprocessor requests.

Reply:
is there any side-effect of missing read_stats when the channel is full, as this is impossible in the previous unbounded chan?

Reply:
The read_stats are flushed from either the `coprocessor` and `storage` per tick, and the default tick interval in YATP is 1 second. So, there are only 2 messages per second, which I think is okay for this limit. However, the problem might be that the `ReadStats` is large?

Reply:
> is there any side-effect of missing read_stats when the channel is full, as this is impossible in the previous unbounded chan?

I think it's okay, because hotspot regions overwhelm the CPU stats channel.